specify/specify7
# Display collection id and user id in denied error dialog

Fixes #4553

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

Log in as a user with limited permissions
Do something you don't have permission to do
See that dialog contains user id and collection id
